kubernetes-repositories.md

```diff
@@ -1,11 +1,11 @@
 ## Kubernetes Repositories
 
 
-This document attempts to outline a structure for creating and associating github repositories with the Kubernetes project.
+This document attempts to outline a structure for creating and associating github repositories with the Kubernetes project. It also describes how and when
```

Comment:
formatting nit: this line runs over 80, but then splits mid sentence.

Reply:
as above.